alpenhorn/cli/node/clean.py

```diff
@@ -51,6 +52,8 @@ def _run_query(
         Replaces the `now` and `cancel` parameters.
     days:
         Converted to datetime
+    has_file:
+        peewee constraint for ArchiveFileCopy.has_file
```

Comment:
Minor, but it would be nice to show an example of what would be passed here (i.e., Example: ArchiveFileCopy.has_file == "Y")